src/indexPage/index.js

```diff
@@ -20,26 +20,24 @@ const loginCheck = () => {
         oauthLogin(() => {
             $('.show-on-load').show();
             $('.barspinner').hide();
+            window.onload = () => {
```

Comment:
It is better to `return` in the if block.

Reply:
Can you elaborate?